testdata/heartbeat_resp_recovers_from_paused.txt

```diff
@@ -67,12 +67,24 @@ stabilize
 > 1 receiving messages
   2->1 MsgHeartbeatResp Term:1 Log:0/0
   3->1 MsgHeartbeatResp Term:1 Log:0/0
+> 1 handling Ready
+  Ready MustSync=false:
+  Messages:
+  1->2 MsgApp Term:1 Log:1/11 Commit:11
+> 2 receiving messages
+  1->2 MsgApp Term:1 Log:1/11 Commit:11
+> 2 handling Ready
+  Ready MustSync=false:
+  Messages:
+  2->1 MsgAppResp Term:1 Log:0/11
+> 1 receiving messages
+  2->1 MsgAppResp Term:1 Log:0/11
 
 # Sad, replica is still tracked as probing.
 status 1
 ----
 1: StateReplicate match=11 next=12
-2: StateProbe match=11 next=12
+2: StateReplicate match=11 next=12
 3: StateReplicate match=11 next=12
 
 # Need to actually distribute a command to sort things out.
```

Comment:
Fix this comment (in the last commit) to say that now the flow stays in the Replicate state?